docs/conf.py

```diff
@@ -79,7 +80,7 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path .
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', 'pages/reference-architecture', 'some/other/file.txt']
```

Comment:
I'm not sure which files these refer to -- are you planning to put these in in the future?

Reply:
Removed.